### PR TITLE
Add some MolStandardize functionality to the CFFI library

### PR DESCRIPTION
**Pull request by greglandrum**
This seems like useful functionality to have available via CFFI

It will be easy to also add support to the JS wrappers, but that needs to wait until all of the MolStandardize code works without needing default parameters from disk. That will be a separate PR.

### CODE REVIEW

File: Code/GraphMol/MolStandardize/MolStandardize.cpp [hunk 1]
```diff
@@ -21,11 +21,42 @@
...
+#define PT_OPT_GET(opt) params.opt = pt.get(#opt, params.opt)
+void updateCleanupParamsFromJSON(CleanupParameters &params,
+                                 const std::string &json) {
+  if (json.empty()) {
+    return;
+  }
+  std::istringstream ss;
+  ss.str(json);
+  boost::property_tree::ptree pt;
+  boost::property_tree::read_json(ss, pt);
```
Comment (bp-kelley) [inline Code/GraphMol/MolStandardize/MolStandardize.cpp:43]: What happens if the json is malformed?

**Review by bp-kelley — APPROVED** [review]

File: Code/GraphMol/MolStandardize/MolStandardize.cpp (hunk 1 above)
Comment (bp-kelley) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp:43]: I would like to add a test for this, but otherwise it looks fine.

Comment (greglandrum) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp:43]: I can add a test, and it would be interesting to know, but the JSON parsing is boost code.

Comment (greglandrum) [reply Code/GraphMol/MolStandardize/MolStandardize.cpp:43]: ok, so what happens with malformed input is that boost::property_tree throws a `json_parser_error` which is derived (eventually) from `std::runtime_error`.

On the Python side this ends up being translated to a `RuntimeError`:
```
In [3]: Draw.UpdateDrawerParamsFromJSON(d2d,"{foo:")
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-2a56b06ecb22> in <module>
----> 1 Draw.UpdateDrawerParamsFromJSON(d2d,"{foo:")

RuntimeError: <unspecified file>(1): expected key string
```
This doesn't seen unreasonable to me, but we could consider doing a PR to modify all of the RDKit code which uses boost::property_tree to throw a `ValueErrorException` instead.

**Pull request merged**